test/boost/memtable_test.cc

```diff
@@ -1170,3 +1173,166 @@ SEASTAR_TEST_CASE(flushing_rate_is_reduced_if_compaction_doesnt_keep_up) {
     });
 }
 
+static future<> exceptions_in_flush_helper(std::unique_ptr<sstables::file_io_extension> mep, bool& should_fail, const bool& did_fail) {
+    auto ext = std::make_shared<db::extensions>();
+    auto cfg = seastar::make_shared<db::config>(ext);
+
+    ext->add_sstable_file_io_extension("test", std::move(mep));
+
+    co_await do_with_cql_env([&](cql_test_env& env) -> future<> {
+
+        co_await env.execute_cql(fmt::format("create table t0 (pk text primary key, v text)"));
+
+        should_fail = true;
+
+        int i = 0;
+
+        BOOST_TEST_MESSAGE("Wait for fail");
```

Comment:
suggestion: we have `testlog`, which provides a nicer end-user experience than `BOOST_TEST_MESSAGE`.

Reply:
Should we not redirect one to the other? I personally tend to use the "-l" option on boost tests quite a bit to know where I am...

Reply:
Changed

Reply:
This is a matter of taste I suppose, that is why I prefixed my comment with "suggestion". I personally prefer the logger.